docker/Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN apt-get -qy update && \
 
 RUN mkdir -p /srv/content-cache
 
-COPY entrypoint.sh /srv/content-cache
+COPY docker/entrypoint.sh /srv/content-cache
```

Comment:
You'll probably have to move this dockerfile to the root directory for the CI to work